charts/mlrun-ce/override-full.yaml

```diff
@@ -24,6 +24,8 @@ mlrun:
         value: v3.2.1-mlk
       - name: MLRUN_KFP_URL
         value: http://ml-pipeline.mlrun.svc.cluster.local:8888
+      - name: MLRUN_CE__MODE
+        value: full
```

Comment:
You added the value `ce.mode` which gets into the config map. So instead of setting this in the extra env, you can change the value in this file to full.